Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -63,6 +83,8 @@ internal static void MoveInputManagerAssetActionsToProjectWideInputActionAsset()
                             AssetDatabase.RemoveObjectFromAsset(obj);
                         }
                     }
+
+                    AssetDatabase.SaveAssets();
```

Comment:
Great - this made the InputManager.asset file change at this point so we could see it cleaned up